core/src/sharedTestFunctions/java/org/kiwix/sharedFunctions/TestModelFunctions.kt

```diff
@@ -164,3 +165,40 @@ fun libraryNetworkEntity(books: List<Book> = emptyList()) = LibraryNetworkEntity
 
 fun recentSearchEntity(id: Long = 0L, searchTerm: String = "", zimId: String = "") =
   RecentSearchEntity(id, searchTerm, zimId)
+
+fun bookOnDiskEntity(
+  id: Long = 0,
+  file: File = File(""),
+  bookId: String = "",
+  title: String = "",
+  description: String = "",
+  language: String = "",
+  creator: String = "",
+  publisher: String = "",
+  date: String = "",
+  url: String? = "",
+  articleCount: String = "",
+  mediaCount: String = "",
+  size: String = "",
+  name: String? = "",
```

Comment:
@macgills What is the difference between `title` and `name`?

Reply:
![nameVsTitle](https://user-images.githubusercontent.com/56556919/92213389-15c3d880-eeb1-11ea-8da0-7ac843e3d739.jpeg)

Is `Test` here the _title_ and `Zim file for testing purpose` the _name_?

Reply:
The title is the display title for a book, the name is a persistent name for zim across revisions if memory serves.
That 2nd line is the Description, look at LibraryViewHolder or BookOnDiskViewHolder to see how we bind the info to list items